### PR TITLE
add shuffle

### LINKED ISSUE #5: Add shuffle button
Allow shuffle of letters to make it easier to think of new words

### PR DESCRIPTION
**Pull request by manasi-m**
Solves #5

### CODE REVIEW

File: app/spellingbee.py [hunk 1]
```diff
@@ -47,22 +47,25 @@ def bee():
...
+            elif word in session['wordsfound']: error = "Already found this word!"
+            elif len(word) < 4: error = "Word must be at least 4 letters long!"
+            elif word.lower() in words:
+                score_calc = score(word, letters)
+                session['score'] += score_calc
+                session['ranking'] = ranking(session['score'], session['maxscore'])
+                session['wordsfound'].append(word)
+            else: error = "Not a word!"
+            
+        if request.form['action'] == 'Shuffle':
```
Comment (rithvikrao) [inline app/spellingbee.py:66]: make this `elif`?

File: app/templates/base.html [hunk 2]
```diff
@@ -33,7 +33,6 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-    <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
```
Comment (rithvikrao) [inline app/templates/base.html:36]: why?

**Review by rithvikrao — CHANGES_REQUESTED** [review]
basically good, just want to confirm things I commented about

File: app/templates/base.html (hunk 2 above)
Comment (manasi-m) [reply app/templates/base.html:36]: there's no style.css file

File: app/spellingbee.py (hunk 1 above)
Comment (manasi-m) [reply app/spellingbee.py:66]: lol fine

File: app/templates/base.html (hunk 2 above)
Comment (rithvikrao) [reply app/templates/base.html:36]: hahah oops that's hilarious

**Pull request merged**